src/praisonai/praisonai/cli/legacy/interactive_legacy.py

```
@@ -1264,8 +1304,30 @@ def interactive_approval_callback(function_name, arguments, risk_level):
+                        streamed = False
+                        stream_deltas = worker_state.get('stream_deltas')
+                        try:
+                            worker_state['stream_header_shown'] = False
+                            worker_state['stream_active'] = True
+                            for chunk in agent.iter_stream(prompt):
+                                if chunk:
+                                    response_str += chunk
+                                    if stream_deltas is not None:
+                                        stream_deltas.put(chunk)
+                            streamed = True
+                        except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)